src/tests/multihost/ipa/test_subid_ranges.py

```diff
@@ -149,18 +141,17 @@ def test_list_subid_ranges(self, multihost):
             2. Should succeed
         """
         ipa_subid_find(multihost)
-        ssh1 = SSHClient(multihost.client[0].ip,
-                         username=user, password=test_password)
+        multihost.client[0].run_command(
+            f'su -l {USER} -c "whoami"', raiseonerr=False)
         cmd = multihost.client[0].run_command(f"cd /tmp/; "
                                               f"./list_subid_ranges "
```

Comment:
@aborah-sudo, I think this should be replaced by `getsubids` from `shadow-utils-subid` package.

Reply:
I tried to keep the code as close as possible to the original one. 
If there is standalone piece of code for getting subid ranges I am not gonna replace it without knowing why.

Reply:
Test was compiling a tool (`./list_subid_ranges`) from sources because it wasn't available by that time.
Now this tool is packaged, so there is no need to compile it on the fly.
This is better to do as a separate PR (CC: @aborah-sudo)

Reply:
Perferct